DataFormats/FWLite/test/test.cppunit.cpp

```diff
@@ -339,51 +339,36 @@ void testRefInROOT::testTwoGoodFiles() {
 }
 
 void testRefInROOT::testGoodChain() {
-  /*
-  TChain eventChain(edm::poolNames::eventTreeName());
-  eventChain.Add((tmpdir + "goodDataFormatsFWLite.root").c_str());
-  eventChain.Add((tmpdir + "good2DataFormatsFWLite.root").c_str());
+  std::vector<std::string> files{(tmpdir + "goodDataFormatsFWLite.root").c_str(),
+                                 (tmpdir + "good2DataFormatsFWLite.root").c_str()};
```

Comment:
Would
```suggestion
  std::vector<std::string> files{tmpdir + "goodDataFormatsFWLite.root",
                                 tmpdir + "good2DataFormatsFWLite.root"};
```
work?

Reply:
Cut & paste from elsewhere in the test, fixed and fixed the other instances too